mirage/apt/apt_inputs.py

```diff
@@ -338,6 +341,32 @@ def create_input_table(self, verbose=False):
         ascii.write(Table(self.exposure_tab), self.output_csv, format='csv', overwrite=True)
         print('csv exposure list written to {}'.format(self.output_csv))
 
+
+    def check_aperture_override(self):
+        if bool(self.exposure_tab['FiducialPointOverride']) == True:
```

Comment:
Apparently, the proper way to check for booleans is to use `if bool(var) is True` 
(https://stackoverflow.com/questions/27276610/boolean-identity-true-vs-is-true)